logging/logger_registry_test.go

```diff
@@ -13,103 +12,73 @@ var (
 	l3 = NewLogger("logger-3")
 )
 
-func mockRegistry() *loggerRegistry {
-	manager := newLoggerRegistry()
```

Comment:
`manager` -> `registry`. In an old version of configurable logging, registries were called "managers". No longer.